interface/templates/interface/subs_for_user.html

```diff
@@ -18,18 +18,23 @@
       <h1>{{ assignment.course.name }} &mdash; {{ assignment.name }}</h1>
       <h2>{{ user }}</h2>
 
-      <table class="table table-striped table-bordered table-sm">
+      <a class="btn btn-primary" href="{% url 'homepage' %}"> Homepage </a>
+      <a class="btn btn-primary" href="{% url 'subs_for_assignment' assignment.course.code assignment.code %}"> Results for {{ assignment.name }} </a>
+
+      <table class="table table-striped table-bordered table-sm mt-2">
         <thead>
           <tr>
             <th scope="col">Time</th>
             <th scope="col">Score</th>
             <th scope="col">Status</th>
+            <th scope="col">Submission id</th>
           </tr>
         </thead>
         <tbody>
           {% for sub in submissions %}
           <tr>
-            <td>{{ sub.timestamp }} </td>
+            <td>{{ sub.timestamp|date:'Y-m-d H:i' }} </td>
+            <td><a href="{% url 'submission_result' sub.id %}"> {{ sub.id}} </a></td>
```

Comment:
Please be consistent with the whitespace. We're using one space inside template tags (`{{ sub.id }}`) and no space inside HTML tags (`<a>{{ sub.id }}</a>`). That extra space in HTML tags is not always ignored by the browser so let's avoid it.